ios/titanium.xcconfig

```diff
@@ -4,7 +4,11 @@
 // OF YOUR TITANIUM SDK YOU'RE BUILDING FOR
 //
 //
+<<<<<<< HEAD
 TITANIUM_SDK_VERSION = 7.5.2.GA
+=======
+TITANIUM_SDK_VERSION = 8.1.0.GA
+>>>>>>> aa52d02... build: use sdk 8.1.0.GA to build
```

Comment:
Merge conflict

Reply:
@hansemannn You're not supposed to see my mistakes! 😃